qemu/tests/win_virtio_driver_installer_uninstall.py

```diff
@@ -98,6 +99,9 @@ def run(test, params, env):
         inf_name = re.findall(r"\.inf", output, re.I)
         if inf_name:
             uninstalled_device.append(device_name)
+            ver_list = win_driver_utils._pnpdrv_info(session, device_name,
+                                                     ["DriverVersion"])
+            test.log.info("Driver version is %s" % ver_list)
```

Comment:
```suggestion
            test.log.info("%s's driver version is %s" % (device_name, ver_list))
```
It should be more clear as it will print device and device's driver info together.

Reply:
@leidwang Updated.